### ISSUE TITLE
Question around what happens when servers are decommissioned

### ISSUE THREAD

**Issue by dil-ddsouza** [tenable/integration-jira-cloud#48]
Hi Steve,

If we were to decommission a server and remove it from the agent group in Tenable.io. How will the script handle the vulnerabilities associated with the decommissioned server? Will it mark it as done? I don't suppose it will remove the vulnerabilities from Jira. 

Thanks,

**Comment by SteveMcGrath** [comment]
hmm, you are correct for IO.  Thats an unfortunate oversight.

**Comment by SteveMcGrath** [comment]
Feel free to test that.  It seems to be working with the assets i manually deleted.

**Comment by dil-ddsouza** [comment]
Hey Steve,

Thank you for getting back, I tried this out but it didn't work i.e. didn't close the vulnerabilities opened in Jira. I downloaded your edited version of Transform.py and ran the integration. 

Is this because I had already ran the script after removing the agents/servers in Tenable.io? So the script kinda ignored the processed ones? 

Also, I had manually changed the status of the vulnerbilities to 'Done' using bulk change and then changed them back to 'To Do' so that I could test the script. 

Thanks
